src/components/SupportTable/SupportTable.jsx

```diff
@@ -1,49 +1,109 @@
-							</td>
-							<td className="govuk-table__cell">{helpRequest.callbackRequired && <>Callback</>}</td>
-							<td className="govuk-table__cell" data-testid="support-requested-table-calls-count">{helpRequest.helpRequestCalls?.length}</td>
-							<td className="govuk-table__cell govuk-table__cell--numeric"  data-testid={`support-requested-table-view_link-${index}`}>
-								<Link
-									href="/helpcase-profile/[resident_id]/manage-request/[help_request]"
-									as={`/helpcase-profile/${helpRequest.residentId}/manage-request/${helpRequest.id}`}>View</Link>
-							</td>
-						</tr>})}
-					</tbody>
-				</table>
+		<>
+			<div className="govuk-tabs" data-module="govuk-tabs">
+				<ul className="govuk-tabs__list">
+					<li className="govuk-tabs__list-item govuk-tabs__list-item--selected">
+						<a className="govuk-tabs__tab" data-testid='support-requested-tab' href="#past-day">
+							Support Requested
+						</a>
+					</li>
+					<li className="govuk-tabs__list-item">
+						<a className="govuk-tabs__tab" data-testid='support-received-tab' href="#past-week">
+							Support Received
+						</a>
+					</li>
+				</ul>
+				<div className="govuk-tabs__panel" id="past-day">
+					<table className="govuk-table">
+						<thead className="govuk-table__head">
+							<tr className="govuk-table__row">
+								<th scope="col" className="govuk-table__header">Type</th>
+								<th scope="col" className="govuk-table__header">Action required</th>
+								<th scope="col" className="govuk-table__header">Call attempts</th>
+								<th scope="col" className="govuk-table__header"></th>
+							</tr>
+						</thead>
+						<tbody className="govuk-table__body">
+							{helpRequests.map((hr, index) => {
+								let latestCallOutcome 
+								let callAttempts = 0
+								if(hr.callbackRequired == true){
+									if(hr.helpRequestCalls.length > 0){
+										let latestCallOutcomeArray = []
+										if(hr.helpRequestCalls[hr.helpRequestCalls.length-1].callOutcome && hr.helpRequestCalls[hr.helpRequestCalls.length-1].callOutcome.includes(",")){
+											let callOutcomesList = hr.helpRequestCalls[hr.helpRequestCalls.length-1].callOutcome.split(",")
+											callOutcomesList.forEach(callOutcome => {
+												if(callOutcomes[callOutcome]){
+													latestCallOutcomeArray.push(" "+callOutcomes[callOutcome]) 
+												}else {
+													latestCallOutcomeArray.push(callOutcome) 
+												}
+											})
+										}else {
+											latestCallOutcomeArray = [callOutcomes[hr.helpRequestCalls[hr.helpRequestCalls.length-1].callOutcome]] || [hr.helpRequestCalls[hr.helpRequestCalls.length-1].callOutcome]
+										}
+										latestCallOutcome = latestCallOutcomeArray.join(",")
+									}else if(hr.helpNeeded.toLowerCase() == 'contact tracing' || hr.helpNeeded == CEV){
+										latestCallOutcome = 'Call required'
+									}
+									else{
+										latestCallOutcome = 'Followup required'
+									}
```

Comment:
Not sure if it's relevant in this case and wether it's the right way to do it, but maybe this can be moved out into the gateway and the result assigned to a field, like the `unsuccessfulCallAttempts` in callbacks gateway?

Reply:
Maybe not necessary in this PR